frontend/src/components/feed/NoticeThread/NoticeThread.styled.ts

```diff
@@ -1,101 +1,161 @@
 import { noticeThreadBackground } from '~/assets/png';
-import type { ThreadSize } from '~/types/size';
+import type { NoticeSize } from '~/types/size';
 
-interface BoardProps {
-  threadSize: ThreadSize;
-  isExpanded: boolean;
-  height: number;
-}
+export const Container = styled.div<{ noticeSize: NoticeSize }>`
+  position: sticky;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 1;
 
-export const Container = styled.div`
-  position: relative;
-`;
+  padding-top: 10px;
 
-export const InnerContainer = styled.div<BoardProps>`
-  overflow: hidden;
-  display: flex;
-  flex-direction: column;
-  row-gap: 20px;
-  position: relative;
-
-  width: 100%;
-  min-height: 200px;
-  max-height: ${({ height }) => height}px;
-  ${({ threadSize }) => {
-    if (threadSize === 'md')
+  background-color: ${({ theme }) => theme.color.GRAY100};
+  border-bottom: 2px solid ${({ theme }) => theme.color.PRIMARY200};
+  transition: 0.3s;
+
+  ${({ noticeSize }) => {
+    if (noticeSize === 'sm')
       return css`
-        padding: 30px 40px;
-        border-radius: 40px;
+        height: 80px;
       `;
-    if (threadSize === 'sm')
+    if (noticeSize === 'md')
       return css`
-        padding: 16px 20px;
-        border-radius: 20px;
+        height: 140px;
+      `;
+    if (noticeSize === 'lg')
+      return css`
+        height: 610px;
       `;
   }}
-  padding-bottom: ${({ isExpanded }) => (isExpanded ? '90px' : '26px')};
+`;
 
-  box-shadow: 0 0 16px ${({ theme }) => theme.color.GRAY300};
+export const BackgroundContainer = styled.div<{ noticeSize: NoticeSize }>`
+  display: flex;
+  border-radius: 20px 20px 0 0;
   background-image: url(${noticeThreadBackground});
   background-size: 100%;
 
-  transition: 0.3s;
+  ${({ noticeSize }) => {
+    if (noticeSize === 'sm')
+      return css`
+        padding: 18px 20px 18px 28px;
+      `;
+    if (noticeSize === 'md')
+      return css`
+        padding: 18px 20px 18px 28px;
+      `;
+    if (noticeSize === 'lg')
+      return css`
+        flex-direction: column;
+        padding: 18px 20px 10px 28px;
+      `;
+  }}
 `;
 
-export const ThreadHeader = styled.header`
+export const InnerContainer = styled.div<{ noticeSize: NoticeSize }>`
   display: flex;
+  ${({ noticeSize }) => {
+    if (noticeSize === 'sm')
+      return css`
+        width: calc(100% - 64px);
+
+        gap: 26px;
+      `;
+    if (noticeSize === 'md')
+      return css`
+        width: calc(100% - 64px);
+        gap: 26px;
+      `;
+    if (noticeSize === 'lg')
+      return css`
+        flex-direction: column;
+        row-gap: 20px;
+      `;
+  }}
+`;
+
+export const ArrowContainer = styled.div`
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+`;
+
+export const ArrowIcon = styled.div<{ disabled: boolean }>`
+  width: 32px;
+  height: 32px;
+  color: ${({ theme }) => theme.color.PRIMARY900};
+  opacity: ${({ disabled }) => (disabled ? 0.3 : 1)};
+`;
+
+export const MegaPhoneWrapper = styled.div`
```

Comment:
```suggestion
export const MegaphoneWrapper = styled.div`
```
메가폰 관련 변수명을 `Megaphone~` 으로 변경하는 걸 제안해 볼게!
왜냐하면, `megaphone` 은 한 단어이니까.